src/Player.js

```diff
@@ -132,15 +132,19 @@ export default class Player {
   playTrack(track) {
     if (!track) return;
 
-    const currentTrack = this.currentlyPlaying?.track || {};
+    const currentTrack = this.currentlyPlaying?.track || { url: null };
```

Comment:
Why is `url` required? `{}.url` would be `undefined`, doesn't that work?

Reply:
It technically does work, my editor settings are just very strict and show warnings when properties are accessed that may not exist.